src/ui/components/Icon/index.js

```diff
@@ -4,18 +4,29 @@ import classNames from 'classnames';
 
 import './styles.scss';
 
+
 export default class Icon extends React.Component {
   static propTypes = {
+    alt: PropTypes.node,
     className: PropTypes.string,
     getRef: PropTypes.func,
     name: PropTypes.string.isRequired,
   }
 
   render() {
-    const { className, name, getRef, ...props } = this.props;
+    const { alt, className, name, getRef, ...props } = this.props;
+
     if (getRef) {
       props.ref = getRef;
     }
-    return <i className={classNames('Icon', `Icon-${name}`, className)} {...props} />;
+
+    // If alt text was included, we'll render that in a hidden span.
+    if (alt) {
+      props.children = <span className="visually-hidden">{alt}</span>;
+    }
+
+    return (
+      <i className={classNames('Icon', `Icon-${name}`, className)} {...props} />
```

Comment:
Ah I didn't realise this compont use used an `i` element under the covers. This makes the alt attribute a little bit weird but I suppose the alt convention works even if it's not actually an alt attribute on an img.